README.md

```diff
@@ -203,362 +516,795 @@ var SendFunds = Exonum.newMessage({
+#### Hexadecimal to Uint8Array
 
-Example of Array of Arrays type declaration:
+```javascript
+var hex = '674718178bd97d3ac5953d0d8e5649ea373c4d98b3b61befd5699800eaa8513b';
 
-```
-var NestedArr = Exonum.newArray({
-    size: 1,
-    type: Exonum.Uint8
-});
-var SomeArr = Exonum.newArray({
-    size: 8,
-    type: NestedArr
-});
-var Type = Exonum.newType({
-    size: 8,
-    fields: {
-        list: {type: SomeArr, size: 8, from: 0, to: 8}
-    }
-});
-var data = {
-    list: [
-        [5, 16, 0],
-        [127, 14]
-    ]
-};
-let buffer = Type.serialize(data)
+Exonum.hexadecimalToUint8Array(hex); // [103, 71, 24, 23, 139, 217, 125, 58, 197, 149, 61, 13, 142, 86, 73, 234, 55, 60, 77, 152, 179, 182, 27, 239, 213, 105, 152, 0, 234, 168, 81, 59]
 ```
 
-## Install
+#### Hexadecimal to String
 
-```
-npm install exonum-client
+```javascript
+var hex = '674718178bd97d3ac5953d0d8e5649ea373c4d98b3b61befd5699800eaa8513b';
+
+Exonum.hexadecimalToBinaryString(hex); // '0110011101000111000110000001011110001011110110010111110100111010110001011001010100111101000011011000111001010110010010011110101000110111001111000100110110011000101100111011011000011011111011111101010101101001100110000000000011101010101010000101000100111011'
 ```
 
-## Build
+#### Uint8Array to Hexadecimal
 
-Install npm dependencies:
+```javascript
+var arr = new Uint8Array([103, 71, 24, 23, 139, 217, 125, 58, 197, 149, 61, 13, 142, 86, 73, 234, 55, 60, 77, 152, 179, 182, 27, 239, 213, 105, 152, 0, 234, 168, 81, 59]);
 
-```
-$ npm install
+Exonum.uint8ArrayToHexadecimal(arr); // '674718178bd97d3ac5953d0d8e5649ea373c4d98b3b61befd5699800eaa8513b'
 ```
 
-Compile browser version into `dist` folder:
+#### Binary String to Uint8Array
 
-```
-$ grunt
+```javascript
+var str = '0110011101000111000110000001011110001011110110010111110100111010110001011001010100111101000011011000111001010110010010011110101000110111001111000100110110011000101100111011011000011011111011111101010101101001100110000000000011101010101010000101000100111011';
+
+Exonum.binaryStringToUint8Array(str); // [103, 71, 24, 23, 139, 217, 125, 58, 197, 149, 61, 13, 142, 86, 73, 234, 55, 60, 77, 152, 179, 182, 27, 239, 213, 105, 152, 0, 234, 168, 81, 59]
 ```
 
-## Tests
+#### Binary String to Hexadecimal
 
-To run tests execute:
+```javascript
+var str = '0110011101000111000110000001011110001011110110010111110100111010110001011001010100111101000011011000111001010110010010011110101000110111001111000100110110011000101100111011011000011011111011111101010101101001100110000000000011101010101010000101000100111011';
 
+Exonum.binaryStringToHexadecimal(str); // '674718178bd97d3ac5953d0d8e5649ea373c4d98b3b61befd5699800eaa8513b'
 ```
-$ grunt test
+
+#### String to Uint8Array
+
+```javascript
+var str = 'Hello world';
+
+Exonum.stringToUint8Array(str); // [72, 101, 108, 108, 111, 32, 119, 111, 114, 108, 100]
 ```
 
+## Community
```

Comment:
IMO, this section does not belong to README; the links are present on the Exonum website.